programming_guide/section-2/section-2c/README.md

```diff
@@ -37,36 +37,36 @@ A data layout transformation is presented as a list of pairs, where each pair re
 ```
 Transformations can be expressed in up to three dimensions on each compute and Shim tile, and in up to four dimensions on Mem tiles. The first element of this array gives the outer-most dimension's stride and size, while the last element of the array gives the inner-most dimension's stride and size. All strides are expressed in <u>multiples of the element width</u>.
 
-Data layout transformations can be viewed as a way to specify to the hardware which location in the data to access next and it is possible to model the access pattern using a series of nested loops. For example, the transformation above can be expressed as:
+> **NOTE:**  The inner-most dimension's stride must be 1 by design.
```

Comment:
@jgmelber This line is part of the AIEOps.td description of `AIE_DMABDOp` but we have examples that do not have a stride of 1 for the inner-most dimension. Does this only apply to DMAs on specific tiles?

Reply:
I think this restriction is relaxed but maybe only for 4B data types? @makslevental

Reply:
Correct https://github.com/Xilinx/mlir-aie/blob/main/lib/Dialect/AIE/IR/AIEDialect.cpp#L1610

Reply:
Thank you! I'll just clarify that in text.